VERSION.dat

```diff
@@ -2,6 +2,6 @@ MAJOR=3
 MINOR=0
 PATCH=12
 PRE_RELEASE_TAG=
-BUILD_METADATA=
-RELEASE_DATE="24 Oct 2023"
-SHLIB_VERSION=3
+BUILD_METADATA=quic
+RELEASE_DATE="24 Sep 2023"
```

Comment:
This should be "24 Oct 2023"

Reply:
Nice catch. Let me know when you've reviewed the rest and I'll push a fixed version.

Reply:
Everything else looks good. This is all I noticed.

Reply:
Fixed version pushed